aten/src/ATen/Parallel.cpp

```diff
@@ -56,7 +92,7 @@ void set_num_threads(size_t nthreads) {
 // region might be different in the new thread;
 // Use init_num_threads() during thread initialization to ensure
 // consistent size of parallel region in different threads
-size_t get_num_threads() {
+int get_num_threads() {
```

Comment:
Might be very detailed, but why not use a fixed bit length integer type? In general I agree with not using an unsigned type